benchmarks/benchmarks/bench_nexus.py

```diff
@@ -858,13 +766,8 @@ def _validate_distributions(
         dist_to_versions: DefaultDict[str, Set[str]] = defaultdict(set)
         for _hub_description, env_to_director in hub_to_director_to_env.items():
             for director in env_to_director.values():
-                accessor = director.make_accessor()
-                for schema in accessor.schemas:
-                    bench_report = json.loads(
-                        accessor.bench_result_file(accessor.get_id(schema)).read_text(),
-                    )
-                    for dist, version in bench_report["pybench_data"]["distributions"].items():
-                        dist_to_versions[dist].add(version)
+                operator = director.make_operator()
+                operator.fill_validation(dist_to_versions)
```

Comment:
Extracting data from pyperf report does not belong to layer of report storing. You can use load_benchmarks method to get all reports